pkg/controller/core.oam.dev/v1alpha2/core/components/componentdefinition/componentdefinition_controller.go

```diff
@@ -108,6 +109,11 @@ func (r *Reconciler) Reconcile(req ctrl.Request) (ctrl.Result, error) {
 			return ctrl.Result{}, util.PatchCondition(ctx, r, &(componentDefinition),
 				cpv1alpha1.ReconcileError(fmt.Errorf(util.ErrCreateOrUpdateDefinitionRevision, defRev.Name, err)))
 		}
+		klog.InfoS("Successfully update DefinitionRevision", "name", defRev.Name)
+
+		if err := coredef.CleanUpDefinitionRevision(ctx, r.Client, &componentDefinition, r.defRevLimit); err != nil {
+			klog.Error("[Garbage collection]")
```

Comment:
Can you please add more information on this error?
And why don't we return the error?

Reply:
this action was align with https://github.com/oam-dev/kubevela/blob/f625877eb1ec1eabfcfabc9668ea4d6da32ab22f/pkg/controller/core.oam.dev/v1alpha2/application/application_controller.go#L202-L206